policies/committer-policy.md

```diff
@@ -75,12 +75,17 @@ or committers with `@openssl/committers`.
 
 ## Commit workflow
 
-We do code reviews on GitHub.  The OpenSSL GitHub repository is a mirror,
-so we do not merge on GitHub.  When you become a committer, we'll send
-you instructions to get commit access to the main repository.  To have
-handy links to review history, we record the reviewers and GitHub pull
-request IDs in commit headers.  We have some helper scripts in the tools
-repo to add these headers automatically.
+Code changes are submitted on Github as pull requests, which are subject to
```

Comment:
CamelCase for "Github"

Reply:
Oh bother.  If you look at line 25, you'll find it spelled "Github" as well.

If it's important, I suggest someone makes a separate "minor edit" PR.

Reply:
Yeah, this can be addressed later.